### DISCUSSION TITLE
ask.puppet.com archive/data migration?

### DISCUSSION THREAD

**Discussion by c33s** [puppetlabs/community#13] [All Things Community]
the ux of breaking links from the google search is not that good. i have this bad experience from time to time since puppet.com restructured it's manual. quite odd to get a 404 where the answer to the question can still partly be seen in the google result.

now the same happened with ask.puppet.com, finding the right answer fast is essential for working with puppet (or other tech stuff). comming from google result where the answer is teasered just to be redirected over a broken certificate (https://www.netlify.com/ ) from ask.puppet.com to the github community with all the data lost which was brought together over many years is simply sad :(

- please keep stable urls (implement redirects)
- mirgate/import data here
- maybe bring back ask.puippet.com in readonly static html with all its content but a banner on top with a link to the new community
- provide search engine friendly content/structure - allow the full content without js, more subpages to not have a big one page containing everything as result after a search.

just my 2cts.

**Comment by binford2k** [comment]
I do have an archive of the content -- the problem is that so much of it is *woefully* out of date and harmfully inaccurate. We did have a static archive up for several years, but retired that and replaced it with redirects to this discussion board.
